samples/base/CMakeLists.txt

```diff
@@ -1,22 +1,73 @@
-cmake_minimum_required(VERSION 3.1)
-
-# determine the build type
-if(NOT CMAKE_BUILD_TYPE)
-    set(CMAKE_BUILD_TYPE Debug CACHE STRING "Choose the type of build (Debug or Release)" FORCE)
-endif()
+cmake_minimum_required(VERSION 3.6.0)
 
 # project name
 project(base)
 
-# define the executable
-set(EXECUTABLE_NAME "base")
-add_executable(${EXECUTABLE_NAME} main.cpp)
+if(ANDROID_NATIVE_API_LEVEL)
+    # build native_app_glue as a static lib
+    set(APP_GLUE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/android/app/src/main/cpp/native_app_glue)
+    include_directories(${APP_GLUE_DIR})
+    add_library( app-glue STATIC ${APP_GLUE_DIR}/android_native_app_glue.c)
+
+    # temporary workaround for findLug not working with gradle
+    set(LUG_DIR ${CMAKE_CURRENT_SOURCE_DIR}/android/app/src/main/lugdunum)
+    include_directories(${LUG_DIR}/include)
+
+    add_library( # Sets the name of the library : must match the name defined in AndroidManifest.xml
```

Comment:
No space after `(`. I doubt of the usefulness of those comments except the one for `AndroidManifest.xml`.